app/policies/contest_relation_policy.rb

```diff
@@ -28,5 +28,9 @@ def update_extra_time?
     return false if record.contest.ended?
     supervise?
   end
+
+  def update_school?
+    user.is_admin?
```

Comment:
Does this give permissions to super admins as well or just admins specifically?

Reply:
It checks for both `:admin` and `:superadmin`
https://github.com/NZOI/nztrain/blob/84633ca11a3cac7d7d6fb9dffa885f8a90350b24/app/models/user.rb#L120-L121

Reply:
Sweet